### ISSUE TITLE
How to set the self.margin?

### ISSUE THREAD

**Issue by Cluoyao** [harveyslash/Facial-Similarity-with-Siamese-Networks-in-Pytorch#25]
Hi, i'm working on this siamese-networks with resnet50, and a new big dataset(the dataset is correct). Now, i have a problem, my loss function converges badly. my self.margin = 1.0, to be honest, i don't know how to set this, Is this parameter important? Do you have any advice? thx!

**Comment by harveyslash** [comment]
margin is a hyperparameter, and unfortunately, I dont know an easy way of finding the best one
